### DISCUSSION TITLE
Zig-Zag Pattern in SAF?

### DISCUSSION THREAD

**Discussion by parkaneric** [GfSE/SAF-Specification#28] [Q&A]
A question came up when we discussed the VPs System Functional Breakdown whether or not we have a functional tree down through several system layers (system -> sub-system -> module):

Should SAF concept highlight derived requirements in the next layer below from system architecture decisions made?
Example: VP SAF_LogicalInternalExchange defines the exchange between elements within SoI.
Thus, in the sub-system layer a requirement has to be elicited to adress and ensure this architecture decision. An according traceability is required which is not available in SAF concept yet.

**Comment by hugoormo** [comment]
In my opinion the Zig-Zag or Zee-Model brings a way to define the architecture in a procedural structured manner. In the other side, the viewpoints often consider the system across the system levels. I would consider the SAF and the Zee-Model two different things, that combine well.

**Comment by haarer** [comment]
In other viewpoints(discussing examples) we had often the comments, that viewpoints should be scoped to only one level (e.g. showing only an element and the sub elements of the system structure.)

Showing a full decomposition (functional or structural  )  might be driven by different concern than the scoped view.

**Comment by haarer** [comment]
I understand the original question was about how capture and trace requirements that are reasoned by architecture decisions instead of being reasoned by upper level functional requirements or non functional requirements.

E.g. the decision to have a system battery powered might require a battery compartment, and the decision to have it accu powered might require a charging port. 
 @parkaneric   could you pls clarify ?

**Comment by parkaneric** [comment]
Many thanks for the respones. Indeed, the requirements capturing concept was misunderstood. Happy to close the discussion.

**Comment by parkaneric** [comment]
Many thanks for the respones. Indeed, the requirements capturing concept was misunderstood. Happy to close the discussion.
